src/functions.php

```diff
@@ -62,10 +51,6 @@ function createDefaultDriver(): FilesystemDriver
         return new EioFilesystemDriver($driver);
     }
 
-    if (\defined("AMP_WORKER")) { // Prevent spawning infinite workers.
-        return new BlockingFilesystemDriver;
-    }
-
     return new ParallelFilesystemDriver;
```

Comment:
How do we prevent infinite ParallelFilesystemDriver instances now?

Reply:
We don't, so tasks can use async file access. @azjezz mentioned this was an issue.

Reply:
No no, [FileTask](https://github.com/amphp/file/blob/6465251f22cb688159a075c9d4f6f477fe108ea6/src/Internal/FileTask.php) which handles file system calls will use the blocking driver for the call. If the worker executes a user task, then another pool of workers might be spawned. I don't see infinite workers being likely (unless someone executes a Task within a Task, but that would be strange), but each worker having its own set of workers would be a possibility. Since by default we limit the number of workers to 8, the resource usage should still be within reason even on systems without eio or uv.